templates/centos/lsyncd.conf.lua.erb

```diff
@@ -3,9 +3,18 @@
 settings {
   logfile = "<%= node['lsyncd']['log_file'] %>",
   statusFile = "<%= node['lsyncd']['status_file'] %>",
-  statusInterval = <%= node['lsyncd']['interval'] %>
+  maxDelays = <%= node['lsyncd']['max_delays'] %>,
+  maxProcesses = <%= node['lsyncd']['max_processes'] %>,
+  statusInterval = <%= node['lsyncd']['interval'] %>,
+  insist = <%= node['lsyncd']['insist'] %>,
 }
 
 -- Hacky way of doing conf.d style configs
-package.path = "/etc/lsyncd/conf.d/?.lua;" .. package.path
-local f = io.popen("ls /etc/lsyncd/conf.d/*.lua|xargs -n1 basename|sed 's/.lua//'") for mod in f:lines() do require(mod) end
+local confdir = '<%= node['lsyncd']['conf_d'] %>/'
+local entries = readdir( confdir )
+
+for name, isdir in pairs( entries ) do
+    if not isdir then
+        dofile( confdir .. name )
+    end
+end
```

Comment:
Hello @andreiocriteo,

It seems to have to different issue here (linked to the https://github.com/axkibe/lsyncd/issues/441) could you split this PR in two commits or create to PR.